TACC/Frontera-Portal
# task/FP-776: Onboarding review action items

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [FP-776](https://jira.tacc.utexas.edu/browse/FP-776)

## Summary of Changes: ##

- Dom Sanitize HTML rendering
- Pending steps show as "disabled"
- Access step now defaults to pending (to allow previous steps to trigger a transition to USERWAIT)
- Admin actions to bypass an dreset steps

## Testing Steps: ##
1. Delete your user
2. Make sure you see a grayed out UI for the Access step while MFA is processing (happens quickly)
3. Make your self staff
4. Make sure you see the admin actions

## UI Photos:

## Notes: ##
